ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml

```diff
@@ -7014,120 +7014,6 @@ periodics:
     - name: release-pull-secret
       secret:
         secretName: ci-pull-credentials
-- agent: kubernetes
-  cluster: api.ci
-  decorate: true
-  decoration_config:
-    skip_cloning: true
-  interval: 6h
-  labels:
-    job-env: ovirt
-    job-release: "4.7"
-    pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: release-openshift-ocp-installer-e2e-ovirt-4.7
```

Comment:
They seem to be referenced elsewhere, e.g.:

```console
$ git grep release-openshift-ocp-installer-e2e-ovirt-4.7
ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:  name: release-openshift-ocp-installer-e2e-ovirt-4.7
core-services/template-deprecation/_allowlist.yaml:        release-openshift-ocp-installer-e2e-ovirt-4.7:
core-services/testgrid-config-generator/_allow-list.yaml:release-openshift-ocp-installer-e2e-ovirt-4.7: informing
```

Reply:
Fixed and another problem I found :)